### PR TITLE
mysql8.0 test: split into build and test jobs

### PR DESCRIPTION
**Pull request by kenhys**
When MROONGA_RELEASE_REQUIREMENT=yes is enabled, at least, minimum
test case must be passed.

### CODE REVIEW

**Comment by kenhys** [comment]
These jobs are aimed to check the following:

1.) env mysql-8.0: 
    execute all tests, so it always fails until all tests are fixed for MySQL 8.0
2.) env mysql-8.0 MROONGA_RELEASE_REQUIREMENT=yes 
   execute some tests which are currently supported. so the job will be succeeded anytime)
  (It is useful to check whether  build task is not broken before releasing new one)

**Comment by kou** [comment]
> 1.) env mysql-8.0:
> execute all tests, so it always fails until all tests are fixed for MySQL 8.0

It's not acceptable. Having any always failing job isn't healthy.

**Comment by kenhys** [comment]
So, I'll unify jobs into 2.)

File: tools/travis/script.sh [hunk 1]
```diff
@@ -102,6 +102,38 @@ run_sql_test()
   case "${CI_MYSQL_VERSION}" in
     mysql-8.0)
       n_processors=1
+      # Remove not supported functionality test
```
Comment (kou) [inline tools/travis/script.sh]: Could you also add TODO comment to remove these `rm`s as soon as possible?

Comment (kenhys) [reply tools/travis/script.sh]: Thanks, I've added comment.

**Pull request merged**